pkg/generators/snippets/deepcopy.go

```diff
@@ -22,9 +24,27 @@ func (in *$.type|raw$) DeepCopy() *$.type|raw$ {
 
 // DeepCopyInto is an autogenerated function
 func (in *$.type|raw$) DeepCopyInto(out *$.type|raw$) {
-	in.$.type|raw$.DeepCopyInto(&out.$.type|raw$)
-}
+	in.$.type|raw$.DeepCopyInto(&out.$.type|raw$)`
```

Comment:
I tested this in by running codegen in kanopy and it works. But I'm curious why would this line still be needed. I would have thought the for loop below would cover this.

For example with APIService, it should iterate through both the members `APIService` and `StructField` and see both have DeepCopyInto methods. I'm not sure why it doesn't see that `APIService` also has [DeepCopyInto](https://github.com/kubernetes/kube-aggregator/blob/master/pkg/apis/apiregistration/v1beta1/zz_generated.deepcopy.go#L29).